### DISCUSSION TITLE
How to enable BasePath with Next-intl/middleware?

### DISCUSSION THREAD

**Discussion by igorgnot** [amannn/next-intl#561] [Q&A]
my basePath in config is /client

My middleware with int is

  const defaultLocale = request.headers.get('x-default-locale') || 'en';
  const handleI18nRouting = createIntlMiddleware({
    locales: ['en', 'pl', 'de'],
    defaultLocale,
    localePrefix: 'always'
  }); 

  const response = handleI18nRouting(request); 

when ask for /client/en/login i get /en/login response
how to fix it?

**Comment by amannn** [comment]
This is tracked in https://github.com/amannn/next-intl/issues/243
